config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/confidential-containers/operator
-  newTag: v0.12.0
+  newName: controller
```

Comment:
Should the newName be reverted to `quay.io/confidential-containers/operator` to fix the CI error ?

@mythi @rayanebel ??

Reply:
yes

Reply:
@rayanebel can you please update the commit to revert `newName` and `newTag`?

Reply:
@bpradipt done.